Changelog.md

```diff
@@ -1,3 +1,14 @@
+## 2.6.0 - Unreleased
```

Comment:
as we have breaking changes (removed features) we may consider version 3.0.0 instead...

Reply:
That's true. I was only thinking about adding a new command, but breaking change is more correct.